policy/examples/README.md

```diff
@@ -4,649 +4,559 @@ This file presents a series of example [Policy documents](../README.md#policy) f
-      "geographies": [
-        "e3ed0a0e-61d3-4887-8b6a-4af4f3769c14"
-      ],
-      "states": {
-        "available": [],
-        "non_operational": [],
-        "reserved": [],
-        "on_trip": []
-      },
-      "vehicle_types": [
-        "bicycle",
-        "scooter"
-      ],
-      "maximum": 10000
-    },
-    {
-      "name": "Not Disadvantaged Communities",
-      "rule_id": "57d47e74-6aef-4f41-b0c5-79bb35aa5b9d",
-      "rule_type": "count",
-      "rule_units": "devices",
-      "geographies": [
-        "1f943d59-ccc9-4d91-b6e2-0c5e771cbc49"
-      ],
-      "states": {
-        "available": [],
-        "non_operational": [],
-        "reserved": [],
-        "on_trip": []
-      },
-      "vehicle_types": [
-        "bicycle",
-        "scooter"
-      ],
-      "maximum": 3000
-    }
-  ]
-}
 ```
 
 [Top](#table-of-contents)
 
-## Idle Time
+## No Riding
 
-This Policy allows scooters and bikes can be in the public right-of-way for up to three days if rentable, and only one day if not.
+The vehicle should not be in one of these defined areas regardless of status.
 
-File: [`idle-time.json`](idle-time.json)
+File: [`no-riding.json`](no-riding.json)
 
 ```json
+ 
 {
-  "policy_id": "a2c9a65f-fd85-463e-9564-fc95ea473f7d",
-  "name": "Idle Times",
-  "description": "LADOT Idle Time Limitations",
-  "start_date": 1552678594428,
-  "end_date": null,
-  "published_date": 1558389669540,
-  "prev_policies": null,
-  "rules": [
-    {
-      "name": "Greater LA (rentable)",
-      "rule_id": "e1942f2d-e5c7-46c4-94c7-293d4e481ed0",
-      "rule_type": "time",
-      "rule_units": "hours",
-      "geographies": [
-        "b4bcc213-4888-48ce-a33d-4dd6c3384bda"
-      ],
-      "states": {
-        "available": [],
-        "reserved": []
-      },
-      "vehicle_types": [
-        "bicycle",
-        "scooter"
-      ],
-      "maximum": 72
-    },
-    {
-      "name": "Greater LA (non-rentable)",
-      "rule_id": "a7eb28b9-969e-4c52-b18c-4243a96f7143",
-      "rule_type": "time",
-      "rule_units": "hours",
-      "geographies": [
-        "12b3fcf5-22af-4b0d-a169-ac7ac903d3b9"
-      ],
-      "states": {
-        "non_operational": [],
-        "on_trip": []
-      },
-      "vehicle_types": [
-        "bicycle",
-        "scooter"
-      ],
-      "maximum": 24
-    }
-  ]
+ "name": "No Ride Zones", 
+ "policy_id": "d78625e9-5a7f-45ae-afab-18ee946acf8f", 
+ "provider_ids": [], 
+ "description": "The vehicle should not be in one of these defined areas regardless of status.", 
+ "start_date": 1617260400000, 
+ "end_date": null, 
+ "published_date": 1630099948146, 
+ "rules": [
+  {
+   "name": " No Ride Zone", 
+   "rule_id": "a2393d69-18a2-44f6-8467-744313a956ed", 
+   "rule_type": "count", 
+   "rate_recurrence": null, 
+   "rate_amount": null, 
+   "geographies": [
+    "f0313182-1ce6-4b3f-a7ce-87ff90051462"
+   ], 
+   "states": {
+    "available": [], 
+    "non_operational": [], 
+    "reserved": [], 
```

Comment:
@jyeo17 as per how I understood [your comment](https://github.com/openmobilityfoundation/mobility-data-specification/issues/764#issuecomment-1196899464), no riding should not be restricted to the state `on_trip`, shouldn't it?

> You can count vehicles that are on_trip to count the process of riding. This works in most cases as trip telemetry and speed isn't usually measured in real-time.